ktlint-core/build.gradle

```diff
@@ -9,8 +9,16 @@ dependencies {
   api deps.logging
   api deps.logback
 
-  testImplementation deps.junit
+  // Standard ruleset is required for EditConfigLoaderTest only
+  testImplementation project(":ktlint-ruleset-standard")
+
+  testImplementation(deps.junit5) {
+    because 'allows to write and run Jupiter tests'
+  }
   testImplementation deps.assertj
   testImplementation deps.jimfs
-  testImplementation project(":ktlint-ruleset-standard")
+}
+
+test {
+  useJUnitPlatform()
 }
```

Comment:
Use https://github.com/Goooler/ktlint/commit/5253d0480d3997408fba76ae9591f28438d7f4fc this have a try.

Reply:
Thanks. It now works as expected and I learned a bit more about gradle.